src/search/landmarks/landmark_factory_hm.cc

```diff
@@ -38,10 +37,12 @@ static void set_difference(vector<int> &set1, const vector<int> &set2) {
     swap(set1, result);
 }
 
-static bool are_mutex(const VariablesProxy &variables,
-                      const FactPair &atom1, const FactPair &atom2) {
-    return variables[atom1.var].get_fact(atom1.value).is_mutex(
-        variables[atom2.var].get_fact(atom2.value));
+static bool are_mutex(
+    const VariablesProxy &variables, const FactPair &atom1,
+    const FactPair &atom2) {
+    return variables[atom1.var]
+        .get_fact(atom1.value)
+        .is_mutex(variables[atom2.var].get_fact(atom2.value));
```

Comment:
I don't like this at all... I guess an easy solution to not get stuff like this would be to introduce variables for some required intermediate part before the return, so not necessarily an issue with the style.